server/api/rounds.py

```diff
@@ -212,6 +215,22 @@ def cumulative_batch_results(election: Election) -> BatchTallies:
     }
 
 
+def batches_times_sampled(election: Election) -> Dict[Tuple[str, str], int]:
+    sampled_batch_draw_counts = (
+        SampledBatchDraw.query.join(Batch)
+        .join(Jurisdiction)
+        .filter_by(election_id=election.id)
+        .group_by(Jurisdiction.name, Batch.name)
+        .values(
+            Jurisdiction.name, Batch.name, func.count(SampledBatchDraw.ticket_number)
```

Comment:
does having `SampledBatchDraw.ticket_number` in the `count` actually change anything?

Reply:
Seems to not make a difference. I'll remove